creator-node/src/routes/files.js

```diff
@@ -170,165 +173,198 @@ const getCID = async (req, res) => {
     )
   }
 
-  startMs = Date.now()
```

Comment:
Was the rationale for deleting this because it doesn't speed things up significantly?